src/sync.js

```diff
@@ -500,6 +510,9 @@ SyncApi.prototype.retryImmediately = function() {
  * @param {Object} syncOptions
  * @param {string} syncOptions.filterId
  * @param {boolean} syncOptions.hasSyncedBefore
+ * @param {Object} syncOptions.savedSync a saved sync that was persisted by a
```

Comment:
can you start the description on a new line so that you get more than 10 chars wide?

Reply:
this is now outdated